app/assets/stylesheets/top_bar_styling.scss

```diff
@@ -0,0 +1,16 @@
+/* #background {
+  background: url(/assets/Boston_Skyline_Night_BW.png) no-repeat fixed center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+  padding-top: 59.05%;
+}
+ */
+.large-3 {
+  background-color: #600000;
```

Comment:
Color literals like `#600000` should only be used in variable declarations; they should be referred to via variable everywhere else.